### ISSUE TITLE
Aesthetic: Please modify the window in the advanced search 

### ISSUE THREAD

**Issue by exonianp** [pods-framework/pods#6264]
## Description

Please increase the frame so that the frame does not show up when making a selection. 

## Testing instructions

4. See error message '....'

## Screenshots / screencast

![image](https://user-images.githubusercontent.com/6937118/139554903-5dcd89ea-6c7f-408b-bcbc-2ff4171a4cd5.png)

## Other super helpful information

<details>
<summary>WordPress Environment</summary>
  
```
Copy and paste your Tools > Site Health > Info. Use the 'Copy Site Info to Clipboard' and paste the results over this line.
```

</details>

<details>
<summary>Pods Package Export</summary>

```json
Copy and paste your Pods Admin > Migrate: Packages > Export and paste the code over this line.
```

</details>

**Comment by JoryHogeveen** [comment]
Hi @exonianp 

Please provide a description on how we can reproduce your situation so we can test this ourselves.
Also include any custom code needed to reproduce.

**Comment by sc0ttkclark** [comment]
Can you provide a package of your configuration (Pods Admin > Migrate Packages) using the Migrate Packages component (Pods Admin > Components)? That might help us to see if we can reproduce the issue.

Also helpful would be access to your Site Health information (Tools > Site Health > Info Tab > Copy site info to clipboard) would also help us determine if there are any potential conflicts from other plugins or your theme.

**Comment by sc0ttkclark** [comment]
Fixed via d817ab01d15f2d26424bf770de79ee840a7c71d7

**Comment by exonianp** [comment]
Thank you @sc0ttkclark  
I am sorry I just seen the messages. I will check it out on 2.8.4 and will let you know if it is sorted.
